packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -287,38 +306,54 @@ export const ChannelView: React.FC = () => {
             </StyledButton>
           </StyledButtonContainer>
         </TitleSection>
-        <TabsContainer>
-          <StyledTabs
-            selected={isSearching ? -1 : TABS.findIndex((x) => x === currentTab)}
-            initialIndex={0}
-            tabs={mappedTabs}
-            onSelectTab={handleSetCurrentTab}
-          />
-          {currentTab === 'Videos' && (
-            <Search
-              searchInputRef={searchInputRef}
-              isSearchInputOpen={isSearchInputOpen}
-              setIsSearchingInputOpen={setIsSearchingInputOpen}
-              setIsSearching={setIsSearching}
-              search={search}
-              isSearching={isSearching}
-              setCurrentTab={setCurrentTab}
+        <TabsWrapper isFiltersOpen={isFiltersOpen}>
+          <TabsContainer>
+            <StyledTabs
+              selected={isSearching ? -1 : TABS.findIndex((x) => x === currentTab)}
+              initialIndex={0}
+              tabs={mappedTabs}
+              onSelectTab={handleSetCurrentTab}
             />
-          )}
-          {currentTab === 'Videos' && (
-            <SortContainer>
-              <Select
-                size="small"
-                labelPosition="left"
-                disabled={isSearching}
-                value={!isSearching ? sortVideosBy : 0}
-                placeholder={isSearching ? 'Best match' : undefined}
-                items={!isSearching ? SORT_OPTIONS : []}
-                onChange={!isSearching ? handleSorting : undefined}
-              />
-            </SortContainer>
-          )}
-        </TabsContainer>
+            {currentTab === 'Videos' ||
+              (currentTab === 'NFTs' && (
```

Comment:
This condition with parenthesis on `currentTab === 'NFTs'` makes it so video tab has no sorting or search, removing parenthesis should fix it